.github/workflows/binary_artifact.yml

```diff
@@ -128,51 +118,39 @@ jobs:
           .\v.exe -skip-unused -prod -cc msvc cmd\tools\vdoctor.v
       - name: Remove excluded
         shell: msys2 {0}
-        run: |
-           rm -rf .git/
-           rm -rf thirdparty/tcc/.git/
-           rm -rf vc/
-           rm -rf v_old.exe
-           rm -rf vlib/v/tests/bench/gcboehm/*.svg
+        run: rm -rf .git/ thirdparty/tcc/.git vc/ v_old.exe vlib/v/tests/bench/gcboehm/*.svg
       - name: Create archive
         shell: msys2 {0}
         run: |
-           cd ..
-           powershell Compress-Archive v $ZIPNAME
-           mv $ZIPNAME v/
-           cd v/
-# NB: the powershell Compress-Archive line is from:
-# https://superuser.com/a/1336434/194881
-# It is needed, because `zip` is not installed by default :-|
+          cd ..
+          # Use `powershell Compress-Archive`, because `zip` is not installed by default.
+          powershell Compress-Archive v $ZIPNAME
+          mv $ZIPNAME v/
+          cd v/
       - name: Create artifact
         uses: actions/upload-artifact@v4
         with:
           name: windows
           path: ${{ env.ZIPNAME }}
 
   release:
-    name: Create Github Release
+    if: github.ref_type == 'tag'
     needs: [build-linux, build-windows, build-macos-x86_64, build-macos-arm64]
     runs-on: ubuntu-latest
     steps:
-      - name: Get short tag name
-        uses: winterjung/split@v2
-        id: split
-        with:
-          msg: ${{ github.ref }}
-          separator: /
       - name: Create Release
         id: create_release
         uses: ncipollo/release-action@v1
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          tag: ${{ steps.split.outputs._2 }}
-          name: ${{ steps.split.outputs._2 }}
+          tag: ${{ github.ref_name }}
```

Comment:
what is the content of `github.ref_name` ?

Reply:
The tag name. Using `ref_nam`e for release tags for like 15 projects.

Reply:
Or do we create tags for V releases that need some special stripping here?

Reply:
For the tags that are used in V it should be the same. 
If would be formated like `release_with_slash/<tagname>` than `GITHUB_BASE_REF` would be still the better way to get the first part than using the split action.

Reply:
Is it `weekly.2024.17` for `git tag -s -m "releases: weekly.2024.17" weekly.2024.17` ?

Reply:
In that case split splited the ref to get the ref name.

For a ref `GITHUB_REF` and `GITHUB_REF_NAME` can be checked here:
https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables